src/overlays/actors/ovl_En_Hit_Tag/z_en_hit_tag.c

```diff
@@ -29,21 +28,65 @@ const ActorInit En_Hit_Tag_InitVars = {
     (ActorFunc)NULL,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BE21F0 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_PLAYER, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_PLAYER,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_NONE,
+    },
     { 16, 32, 0, { 0, 0, 0 } },
 };
 
-#endif
+void EnHitTag_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ColliderCylinder* pCylinder;
+    EnHitTag* this = (EnHitTag*)thisx;
 
-extern ColliderCylinderInit D_80BE21F0;
+    Actor_SetScale(&this->actor, 1.0f);
+    this->actionFunc = EnHitTag_WaitForHit;
+    pCylinder = &this->cylinder;
+    Collider_InitAndSetCylinder(globalCtx, pCylinder, &this->actor, &sCylinderInit);
+    Collider_UpdateCylinder(&this->actor, pCylinder);
+    if (Flags_GetSwitch(globalCtx, ENHITTAG_GET_SWITCHFLAG(this)) {
+        Actor_MarkForDeath(&this->actor);
+    }
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Hit_Tag/EnHitTag_Init.s")
+void EnHitTag_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnHitTag* this = (EnHitTag*)thisx;
+    Collider_DestroyCylinder(globalCtx, &this->cylinder);
```

Comment:
```suggestion
    EnHitTag* this = THIS;
    
    Collider_DestroyCylinder(globalCtx, &this->cylinder);
```
We like to leave a space between var declarations and code.